Makefile

```diff
@@ -1,10 +1,11 @@
 .DEFAULT_GOAL:=help
 
 VERSION=$(shell git rev-parse HEAD)
+RELEASE_TAG ?= "0.0.0"
 
 .PHONY: build
 build:
-	go build -o preflight -ldflags "-X github.com/redhat-openshift-ecosystem/openshift-preflight/version.commit=$(VERSION)" main.go
+	go build -o preflight -ldflags "-X github.com/redhat-openshift-ecosystem/openshift-preflight/version.commit=$(VERSION) -X github.com/redhat-openshift-ecosystem/openshift-preflight/version.version=$(RELEASE_TAG)" main.go
```

Comment:
Can we move the ldflags to a variable to shorten this line? I can add a separate PR if that helps.

Reply:
Yes.  A separate PR will be great.  Thanks.